addons/web/static/src/js/views/form_relational_widgets.js

```diff
@@ -7,6 +7,7 @@ var data = require('web.data');
 var Dialog = require('web.Dialog');
 var common = require('web.form_common');
 var ListView = require('web.ListView');
+require('web.ListEditor'); // require because list editable because add the start_edition methods
```

Comment:
your comment dont not unmake sense...

seriously, I have no idea what you are trying to say